src/controllers/userController.js

```diff
@@ -88,14 +135,40 @@ const updateUserPassword = async (req, res) => {
 };
 
 const deleteSingleUser = async (req, res) => {
+  /*
+     #swagger.summary = 'Fetch a user by id and delete user.'
+     #swagger.parameters['id'] = {
+        description: 'Mongo ObjectID of the user to fetch',
+     }
+     #swagger.responses[200] = {
+				description: 'User successfully fetched and deleted.'
+				
+		 }
+		  #swagger.responses[404] = { description: 'No user with id found.' }
+  */
   const userId = req.params.id;
+  if (!userId) {
+    // Throw a NotFoundError if the user is not found
+    throw new CustomError.NotFoundError(`No user with id: ${req.params.id}`);
+  }
   // Find the user by ID and delete
   await User.findByIdAndDelete(userId);
   res.status(StatusCodes.OK).json({ message: 'User deleted successfully' });
 };
 
 //Fetching a user from the database, including all the albums they've purchased
 const getCurrentUserWithPurchasedAlbums = async (req, res) => {
+  /*
+     #swagger.summary = 'Fetch a user by id and all albums they purchased.'
+     #swagger.parameters['id'] = {
+        description: 'Mongo ObjectID of the user to fetch',
+     }
```

Comment:
This endpoint does not take any query or path parameters; it just uses the cookie to find the requesting user's id.